UPGRADE-18.0.md

```diff
@@ -0,0 +1,979 @@
+# UPGRADING FROM 17.x to 18.0
+
+The releases of Shopsys Platform adhere to the [Backward Compatibility Promise](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/) to make the upgrades to new versions easier and help long-term maintainability.
+
+## Recommended way of upgrading
+
+Since there are two possible scenarios for using Shopsys Platform, instructions are divided into these scenarios.
+
+### You use our packages only
+
+Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or `shopsys/http-smoke-testing`.
+
```

Comment:
I'm not sure if this is still applicable... I don't think we have sections split